### PR TITLE
Documentation: update contributing guidelines

### LINKED ISSUE #106: Documentation: Remove place holders in contributing guidelines
**Describe the task that needs to be done.**
The contributing guidelines currently has placeholder data from the start of the project. These need to be populated with current information.

**Describe how a solution to your proposed task might look like (and any alternatives considered).**
Complete contributing guidelines

**Notes**
Clean up task before handover

### PR DESCRIPTION
**Pull request by arathibiju**
## Description
Fixes #106. Fixes grammar, spelling, readability and fills out formatting information.

## How has this been tested?
N/A

## Screenshots
N/A

## Checklist
*(Leave blank if not applicable)*

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by arathibiju** [comment]
Apologies I forgot to rebase off main, will do that soon

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,19 +31,19 @@ This project uses the fork and pull workflow. For more details see [this video](
...
 
 ## Code review process
-Team members in the relevant sub-team will review the code submitted for PR. Team members will review code for readability, style, bugs, test coverage and completion of acceptance criteria. The reviwers should also make ensure that the application works as expected, all prevoius tests pass, and new features have associated unit tests.
+Team members in the relevant sub-team will review the code submitted for PR. Team members will review code for readability, style, bugs, test coverage and completion of acceptance criteria. The reviewers should also ensure that the application works as expected. All previous tests pass, and new features have associated unit tests.
 
 ## Code, commit message and branch name conventions
-(When linting and formatting is finalised add details here)
+The IntelliJ IDE is reccomended for the backend to format and refactor code where necessary.
```
Comment (uoa-ypec413) [inline CONTRIBUTING.md:46]: Could you please fix the spelling for this?

```suggestion
The IntelliJ IDE is recommended for the backend to format and refactor code where necessary.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,19 +31,19 @@ This project uses the fork and pull workflow. For more details see [this video](
...
 
 ## Owners
-The different parts of the project are owned by different contributors. The owners of each sub-team should review PR's related to their area of the project. [insert links to owners files] 
+Different contributors own the different parts of the project. The owners of each sub-team should review PRs related to their area of the project.
```
Comment (uoa-ypec413) [inline CONTRIBUTING.md:49]: Could we please link the CODEOWNERS file here?

**Review by uoa-ypec413 — CHANGES_REQUESTED** [review]
LGTM other than a couple of small changes :)

**Review by uoa-ypec413 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (arathibiju) [reply CONTRIBUTING.md:49]: Sure will do

**Pull request merged**